### ISSUE TITLE
Poor capitalization choices in default childViewEventPrefix

### ISSUE THREAD

**Issue by MeoMix** [marionettejs/backbone.marionette#2236]
Hey,

Currently, if I want a CollectionView to listen to a child's event the naming of my function looks like:

    onChildviewFoo: function(){}

I was expecting it to be:

    onChildViewFoo: function(){}

This would be a breaking change, unless both ways are supported, so maybe not until 3.0, but it would be nice to have some consistent capitalization.

**Comment by jmeas** [comment]
https://github.com/marionettejs/backbone.marionette/pull/2206
